src/strands/experimental/bidirectional_streaming/__init__.py

```diff
@@ -4,7 +4,7 @@
 from .agent.agent import BidiAgent
 
 # IO channels - Hardware abstraction
-from .io.audio import AudioIO
+from .io.audio import BidiAudioIO
```

Comment:
This is a pretty generic abstraction. are there any non-bidi use cases? 🤔  should we add the bidi prefix?